public/components/data_connections/components/datasources_header.tsx

```diff
@@ -15,7 +15,7 @@ import _ from 'lodash';
 import React from 'react';
 import { OPENSEARCH_DOCUMENTATION_URL } from '../../../../common/constants/data_connections';
 
-export function DataConnectionsHeader() {
+export const DataConnectionsHeader = () => {
```

Comment:
just curious why this change?

Reply:
Wanted to make it consistent with rest of the observability.